r/tests/testthat/test-dplyr-funcs-conditional.R

```diff
@@ -30,7 +30,7 @@ test_that("if_else and ifelse", {
   compare_dplyr_binding(
     .input %>%
       mutate(
-        y = if_else(int > 5, 1, 0),
+        y = as.integer(int > 5),
```

Comment:
The changes in these tests are incorrect as they completely change what is being tested here